### PR TITLE
stdlib 9 compat

### PR DESCRIPTION
**Pull request by robertc99**
#### Pull Request (PR) description
Ensure compat with latest stdlib

### CODE REVIEW

File: manifests/defaults.pp [hunk 1]
```diff
@@ -16,7 +16,7 @@
   if $create_base_rules {
     $rules.each |$rule_name, $params| {
       if !defined(Logrotate::Rule[$rule_name]) {
-        $_merged_params = merge($rule_default,$params)
+        $_merged_params = stdlib::merge($rule_default,$params)
```
Comment (traylenator) [inline manifests/defaults.pp:19]: Does the lower band of stdlib in metadata need to be changed ?

Comment (robertc99) [reply manifests/defaults.pp:19]: stdlib::merge would require stdlib9. But it occurs to me that a better solution might be to use the + operator and avoid the whole issue.

**Comment by bastelfreak** [comment]
@robertc99 please rebase, https://github.com/voxpupuli/puppet-logrotate/pull/206 fixed the CI errors.

**Comment by bastelfreak** [comment]
@robertc99 please do an actual rebase to get rid of the merge commit.

**Comment by robertc99** [comment]
yeah, Im not sure how to do that. I can close the pull. Refork it and make my change again.

**Pull request closed without merging**